tajo-core/src/test/java/org/apache/tajo/engine/query/TestSelectQuery.java

```diff
@@ -105,6 +105,20 @@ public final void testExplainSelect() throws Exception {
   }
 
   @Test
+  @SimpleTest(queries = {
```

Comment:
Yes, it's for simple tests, as the name implies. Serious test would need some more respect
 (separate test method). And for the second question, would it be enough adding index for fail message something like,
assertEquals("Result Verification for: " + (i+1) + "th test",